### PR TITLE
CLOUDP-66899: mongocli atlas cluster(s) onlineArchive(s) start

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

_Jira ticket:_ [Name of ticket](https://jira.mongodb.org/browse/[name-of-ticket])

## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Further comments

pause 
```zsh
mongocli atlas clusters onlinearchive pause 5f08577acf7bf723975cb4c4 --clusterName testShard
{
	"_id": "5f08577acf7bf723975cb4c4",
	"clusterName": "testShard",
	"collName": "test",
	"criteria": {
		"dateField": "test",
		"expireAfterDays": 3
	},
	"dbName": "test",
	"groupId": "5e429f2e06822c6eac4d59c9",
	"partitionFields": [
		{
			"fieldName": "test",
			"fieldType": "date",
			"order": 0
		}
	],
	"paused": true,
	"state": "PAUSED"
}

```

start
```zsh
mongocli atlas clusters onlinearchive start 5f08577acf7bf723975cb4c4 --clusterName testShard
{
	"_id": "5f08577acf7bf723975cb4c4",
	"clusterName": "testShard",
	"collName": "test",
	"criteria": {
		"dateField": "test",
		"expireAfterDays": 3
	},
	"dbName": "test",
	"groupId": "5e429f2e06822c6eac4d59c9",
	"partitionFields": [
		{
			"fieldName": "test",
			"fieldType": "date",
			"order": 0
		}
	],
	"paused": false,
	"state": "ACTIVE"
}
```

### CODE REVIEW

**Comment by gssbzn** [comment]
Sadly I could not e2e this as you need to have a collection in your cluster for it to work

File: internal/cli/atlas/onlinearchive/pause.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	var err error
+	opts.store, err = store.New(config.Default())
+	return err
+}
+
+func (opts *PauseOpts) Run() error {
+	paused := true
+	cluster := &atlas.OnlineArchive{
+		ID:     opts.id,
+		Paused: &paused,
```
Comment (robcarlan-mlab) [inline internal/cli/atlas/onlinearchive/pause.go:46]: [np] can use the `pointy` library here? (and a couple of other places)

File: internal/cli/atlas/onlinearchive/pause.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	}
+	result, err := opts.store.UpdateOnlineArchive(opts.ConfigProjectID(), opts.clusterName, cluster)
+
+	if err != nil {
```
Comment (robcarlan-mlab) [inline internal/cli/atlas/onlinearchive/pause.go:50]: [np] could rm the blank line to group these together

File: internal/cli/atlas/onlinearchive/start.go [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+func (opts *StartOpts) Run() error {
+	paused := false
+	archive := &atlas.OnlineArchive{
+		ID:     opts.id,
+		Paused: &paused,
+	}
+	result, err := opts.store.UpdateOnlineArchive(opts.ConfigProjectID(), opts.clusterName, archive)
+
+	if err != nil {
```
Comment (robcarlan-mlab) [inline internal/cli/atlas/onlinearchive/start.go:50]: same [np] w pointy & whitespace

Comment (gssbzn) [reply internal/cli/atlas/onlinearchive/start.go:50]: we don't have pointy as a dependency in the cli, we could discuss adding it in a separate ticket/PR

Comment (robcarlan-mlab) [reply internal/cli/atlas/onlinearchive/start.go:49]: Do we need to jump through any hoops to add new dependencies?

Comment (gssbzn) [reply internal/cli/atlas/onlinearchive/start.go:49]: @robcarlan-mlab as long as it's licence is pre approved by legal no, if is a more restrictive license we need clearance from them. Other than that not that I could think of

**Pull request merged**